envbuilder.go

```diff
@@ -233,60 +89,49 @@ type Options struct {
 type DockerConfig configfile.ConfigFile
 
 // Run runs the envbuilder.
-func Run(ctx context.Context, options Options) error {
-	if options.InitScript == "" {
-		options.InitScript = "sleep infinity"
-	}
-	if options.InitCommand == "" {
-		options.InitCommand = "/bin/sh"
-	}
-	if options.IgnorePaths == nil {
-		// Kubernetes frequently stores secrets in /var/run/secrets, and
-		// other applications might as well. This seems to be a sensible
-		// default, but if that changes, it's simple to adjust.
-		options.IgnorePaths = []string{"/var/run"}
-	}
+func Run(ctx context.Context, options OptionsMap, deps Dependencies) error {
```

Comment:
The idea of extracting `Logger` and `FileSystem` from `Options` is correct, but it is more natural to just keep them as separate arguments, so:

```go
func Run(ctx context.Context, options OptionsMap, logger Logger, filesystem Filesystem) error {
```